backend-api/template.yaml

```diff
@@ -982,6 +982,91 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  AsyncBiometricTokenFunction:
+    Type: AWS::Serverless::Function
+    DependsOn:
+      - AsyncBiometricTokenLogGroup
+    Metadata:
+      BuildMethod: esbuild
+      BuildProperties:
+        Minify: true
+        Target: es2022
+        Sourcemap: false
+        EntryPoints:
+          - src/functions/asyncBiometricToken/asyncBiometricTokenHandler.ts
+    Properties:
+      AutoPublishAlias: live
+      FunctionName: !Sub ${AWS::StackName}-biometric-token
+      Runtime: nodejs20.x
+      Handler: asyncBiometricTokenHandler.lambdaHandler
+      Events:
+        AsyncBiometricToken:
+          Type: Api
+          Properties:
+            Path: /async/biometricToken
+            Method: post
+            RestApiId: !Ref SessionsApi
+      Role: !GetAtt AsyncBiometricTokenLambdaRole.Arn
+      VpcConfig:
+        SubnetIds:
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdA
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdB
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdC
+        SecurityGroupIds:
+          - !ImportValue devplatform-vpc-AWSServicesEndpointSecurityGroupId
+
+  AsyncBiometricTokenFunctionPermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName: !GetAtt AsyncBiometricTokenFunction.Arn
+      Principal: events.amazonaws.com
+      SourceAccount: !Sub ${AWS::AccountId}
+      SourceArn: !Sub arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${SessionsApi}/*/*/*
+
+  AsyncBiometricTokenLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 30
```

Comment:
Almost certainly beyond the scope of this ticket, as I know this is what we do elsewhere, and maybe more for @sandymay-dd  - but should we be retaining logs for 30 days, even in dev/build? That sounds expensive (although come to think if it I can't remember if they end up getting retained for 30 days anyway regardless of what we set here due to the platform setup).

Reply:
I agree in principle. Unfortunately, it seems to be a programme requirement: https://github.com/govuk-one-login/mobile-id-check-backend/pull/1468

Reply:
😢 didn't realise it was policy, thanks for the link - seems an impractical choice for dev but I can see why it'd be hard to discriminate technically